_posts/2024-02-17-survey-https-infra.md

```diff
@@ -0,0 +1,195 @@
+As such, a lot of effort has gone into ensure it is both.
+
+When you first start OpenTTD 14, you are greeted by a dialog asking you if you want to opt-in to the survey.
+If you don't (either close the dialog or decline), no survey information will ever be sent to us.
+But if you do, at the end of every game, a small file is transmitted to us.
+
+What is in it you ask?
+For that, we created a "Preview" button, which shows exactly what information would be sent.
+Nothing more, nothing less.
+What is in there, is what we receive.
+
+And if you change your mind (or want to see the Preview again), under "Game Options" you can change your choice at any time.
+No strings attached; no nag-screens "are you sure?!".
+We highly appreciate it if you enable it.
+But if you don't, that is perfectly fine too.
+
+Once received, the survey is processed, checked whether it is a valid survey, for what version, and stored in an efficient way in the backend.
+Every once in a while, an analyzer runs over all the surveys to produce survey results.
+They can be seen [here](https://survey.openttd.org/summaries), with all their details made available.
+
+There are all kinds of protection against over-representing one population or the other, although we still have work to do to tune that system better.
+But, as more surveys arrive, we also have more information to improve on this.
+
+So if you are curious how your playstyle is versus many other people, check out the summaries and get an impression!
+
+Just be warned, and I have to remind people about this almost daily: the summaries are a snapshot of that moment in time, and can over-represent one group or the other.
+Although as more time will go on, this problem will become smaller and smaller, until the actual 14.0 is released, be aware that the sample size is rather small, and summaries can differ widely from week to week.
+In other words: be careful to not make wrong conclusions on something, based on a result of a single week.
+
+## Popularity
+
+Often it is rumoured that OpenTTD is dead.
+We don't seen any evidence of that, how ever you slice it.
+
+OpenTTD development is doing very well, as the new feature list for 14.0 is showing.
+But also the player-base has not been declining in months.
+
+Take for example the [amount of concurrent players on Steam](https://steamdb.info/app/1536610/charts/#all).
+In this last 3 years this amount has grown, from ~600 to ~1000.
+That is a huge increase.
+
+From internal Steam financial information (yes, free games also have financial information), we can see that the game has been installed by almost 2 million unique users, in the last 3 years.
+That is so many more players than I ever expected would play this game.
+
+More information from Steam, we can also see how popular our news posts are.
+Our 13.0 release post, for example, have been read by 150,000 unique players (with over 2.5M impressions).
+That is a lot, for a free game that is 20 years old!
```

Comment:
May I propose this to be released on the 2023-03-14? Then maybe even add "Yes, today is OpenTTD's birthday!" or something along those lines.

Reply:
But the birthday is on 2014-03-06 :) The 14th is already 0.1.1